charters/charter-template.md

```diff
@@ -0,0 +1,30 @@
+
+# Project Name
+
+
+## Description
+
+## Objective
+
+## In-scope
+
+## Out-of-scope
+
+## Milestones
+
+## Roles
+
+### Project Lead
+
+### Product Owner
+
+### Technical Lead
+
+
+## Communication
```

Comment:
So this is interesting because it formalizes modes of communication. I agree that we need formal channels of communication but wonder if "slack", "mailing lists", and "forums" will always be the case. I am ok with adding under ## Communication a line like "(eg. slack channel(s), mailing list(s), and discussion forum(s))" but making them sections in the template translates to me as formalizes them which I am not sure is the intention.

Reply:
Absolutely correct. It was _shorthand_ on my part because I did not include descriptions for template sections in the sketch. The intent is to capture information to populate the community page as projects are chartered. I'll update in the next commit.